### PR TITLE
VIH-6857 Change to page and button rename

### PR DESCRIPTION
**Pull request by MichaelKeaneathmcts**
### JIRA link (if applicable) ###
VIH-6857

### Change description ###

Changed handling for a renames button & replaced code that handles the page.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs [hunk 1]
```diff
@@ -85,8 +85,9 @@ public void WhenTheyEnterPrivateConsultationRoom()
         {
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeWaitingRoomPage.HearingTitle).Displayed.Should().BeTrue();
             WaitForUserStatusToBe(ParticipantState.Available);
+            Thread.Sleep(1500);  // unfortunately need a hard sleep as the button is active in html but does not respond for another second.
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs:88]: Why not use the `WaitUntilVisible` method instead of sleeping?

Comment (MichaelKeaneathmcts) [reply VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs:88]: WaitUntilVisible did not work, the button **was** visible and also active, however there's a background delay before a click on the button will actually work.

Comment (shaed-parkar) [reply VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs:88]: @MichaelKeaneathmcts it's a 🚨 when thread sleep is dropped across the code. Also when someone enters a consultation room, why not wait for the status to be updated to "InConsultation" and then you know they're actually in the consultation room

**Comment by MichaelKeaneathmcts** [comment]
Application changes require additional changes that have been made to the branch, will submit a new pr for the whole.

**Pull request closed without merging**